### DISCUSSION TITLE
New Car Information

### DISCUSSION THREAD

**Discussion by JollyJumper88** [BimmerGestalt/AAIdrive#784] [Ideas]
Hi, 
first of all, I love this app and also the new car information display with all its data.

for me, I would love to just see the CURRENT GEAR I am in, written in big letters, and nothing more for the track. Would it be possible to just extend this? or maybe a filter to hide all the other stuff. All the info is nice but not actually needed while driving. 

I tried to extend the application myself but struggled with google maps and spotify keys that I do not have.

**Comment by hufman** [comment]
Unfortunately, there is no option to change the font size in these screens.
Making the screens fully customizable is more effort than I'm willing to do currently. The Developer Guide should help with setting up the keys (and also Google Maps is not enabled by default)
